README.md

```diff
@@ -122,55 +74,74 @@ i18n({
-});
-```
-
-### Fallback namespace:
-
-Set the namespace which is used as the fallback when none is specified, or the requested key does not exist in the requested namespace - Default: `'default'`
+var i18n = require('hmpo-i18n');
 
-```javascript
-i18n({
-    fallbackNamespace: 'admin'
+// i18n fires a "ready" event when it is done loading resources
+i18n.on('ready', function () {
+    i18n.translate('welcome.title', 'en');
 });
 ```
 
-If required, both the `fallbackLng` and `fallbackNamespace` options can be passed as an array.
-
-### Resource path
-
-For the fs resource loader (currently the only backend supported), sets the location to load resource files from, and pattern for parsing namespace and language from the file path - Default: `locales/__lng__/__ns__.__ext__`.
+### 2. As an express middleware:
 
 ```javascript
-i18n({
-    path: '/var/i18n/__lng__/__ns__/resource.__ext__'
-});
-```
+var app = require('express')();
 
-### Base directory
+var i18n = require('hmpo-i18n');
 
-The root directory in which to look for resources on the `path` defined - Default: `process.cwd()`
+i18n.middleware(app);
 
-```javascript
-i18n({
-    baseDir: __dirname
+app.use(function (req, res, next) {
+    // load language from querystring parameter - e.g. ?lang=en
+    req.setLanguage(req.query.lang);
+    
+    // a translate method is now available on the request
+    // this will translate keys according to the language request headers
+    res.render('index', {
+        title: req.translate('title')
+        
+        // simple language option
+        name: req.translate('name', 'fr');
+        
+        // simple namespace option
+        message: req.translate('greeting', { name: 'admin' });
+        
+        // multiple langauge options as an array
+        // will look for `en-GB` first, and fall back to `en` only if a translation for `en-GB` is not found.
+        subject: req.translate('deep.object', ['en-GB', 'en']);
+    });
 });
+
+app.listen(3000);
+...
 ```
 
-*Note: if you are using hmpo-i18n inside a module which is likely to be installed as a child dependency of another project then it is highly recommended that you set this property.*
+## Configuration Options
+
+| Option              | Description |
+|---------------------|-------------|
+| `baseDir`           | Root directory for resource lookup based on path. Default: process.cwd(). 
+| `allowedLangs`      | List of allowed languages 
+| `locale`            | Default locale 
+| `fallbackLocale`    | Locale to use if translation is missing 
+| `cookie`            | Cookie settings (e.g., `{ name: 'lang' }`) 
+| `query`             | Name of query parameter 
+| `session`           | Name of session parameter 
+| `noCache`           | Disable caching of resources 
+| `watch`             | Watch for resource file changes 
+| `setHeaders`        | Set Content-Language header 
+| `fallbackLng`       | Sets the fallback language if none is specified or a key is missing. Default: 'en'.
```

Comment:
This should be fallbackLang? I think this is a spelling mistake taken from the original readme